runtime/ast/expression.go

```diff
@@ -377,6 +377,7 @@ type AccessExpression interface {
 type MemberExpression struct {
 	Expression Expression
 	Optional   bool
+	AccessPos  Position
```

Comment:
Minor: Might be good to describe with a comment what this field records!

Reply:
Added a comment in 76a15454cf02cac6fb9ee3ef3f766cff98d4fc77